LOG.tbasicu

```diff
@@ -216,124 +220,123 @@ Type tLogThis
     
       end if
       
-      if Bit_Get(logWhere,5) then ' %LT_ToLogFile is SET
-        if me.RotateLog = %FALSE then allOK = allOK and %FALSE
+      if me.private_isFlagSet(logWhere, %LT_ToLogFile) then
+        if me.RotateLog = %FALSE then allOK = %FALSE
         local sb as string
         sb = sDTF & sTagF & logWhatTitleFile & me.titleSeparatorFile & logWhat & me.endLineFile
-        successFile = FILE_Append(me.logFile,sb)
-        if successFile <> 0 Then allOK = allOK and %FALSE
-        Reset successFile
+        
+        allOK = (FILE_Append(me.logFile,sb) = 0)
```

Comment:
SuccessFile takes value of 0 for success.

That means that allOk if File_Append = 0. And this is what this line does.

In case File_Append would return non zero, allOk will be %FALSE.

Reply:
why it is not `allOK = allOK and (FILE_Append(me.logFile,sb) = 0)` ?  I would like _allOK_ remains _%FALSE_ if it already was and `(FILE_Append(me.logFile,sb) = 0)` were _%TRUE_

Reply:
Aha! That was the motivation to do it this way. I did not get it, will adjust.

Reply:
Fixed, please have a look, thank you 👋

Reply:
Petr, 
**everything but** [line 233 ](https://github.com/DirectuX/logger/pull/3/files#diff-060f642948211dc1e4e83624f5f6cac1R233) is fine now. Tell me if I'm wrong, as-is it would reset line 229 changes or potential previous %_FALSE_ states of _allOK_?

Reply:
```suggestion
        allOK = allOK and (FILE_Append(me.logFile,sb) = 0)
```

Reply:
Addressed the issue :)

Reply:
Thanks a lot for reporting it. Please let me know if there is something more, if not, I think we can merge :)